spec/busted-ci-helper.lua

```diff
@@ -1,43 +1,49 @@
 -- busted-ci-helper.lua
+local busted = require 'busted'
 
+do
+  local shutdown_timers = require("kong.cmd.utils.timer").shutdown
+  assert(type(shutdown_timers) == "function")
 
--- needed before requiring 'socket.unix'
-require 'socket'
+  -- shutdown lua-resty-timer-ng to allow the nginx worker to stop quickly
+  busted.subscribe({ 'exit' }, shutdown_timers)
+end
```

Comment:
FYI I tried putting this into our own `bin/busted` wrapper, but it doesn't work there. `busted` only exposes the `busted.subscribe()` function _within_ code that is called by `busted.runner`, such as the `--helper` script and output handler.